modules/apps/callbacks/types/keys.go

```diff
@@ -0,0 +1,27 @@
+package types
+
+type CallbackType string
+
+const (
+	ModuleName = "ibccallbacks"
+
+	CallbackTypeSendPacket           CallbackType = "send_packet"
+	CallbackTypeAcknowledgement      CallbackType = "acknowledgement"
+	CallbackTypeTimeoutPacket        CallbackType = "timeout"
+	CallbackTypeWriteAcknowledgement CallbackType = "write_acknowledgement"
+
+	// Additional packet data is expected to specify the source callback in the following format
+	// under this key:
+	// {"src_callback": { ... }}
+	SourceCallbackMemoKey = "src_callback"
```

Comment:
I think we can drop the `Memo` reference

Reply:
If we want to document usage of memo in ica and transfer, I think it's okay to be explicit for those apps:

> ICS20 and ICS27 will store the callback packet data in the memo field as a json object. The expected format is as follows:

Reply:
done